www/_template/tutorials/svelte.md

```diff
@@ -0,0 +1,327 @@
+
+```diff
+<!-- public/index.html -->
+
+  <body>
+-   <h1>Welcome to Snowpack!</h1>
+-   <script type="module" src="/index.js"></script>
++   <script type="module" src="/_dist_/index.js"></script>
+  </body>
+```
+
+You'll need to restart Snowpack (stop the process in terminal and then run `npm start` again) for configuration file changes. When you start up again, if it worked, it should look exactly the same since all the changes you just made are behind the scenes.
+
+## Adding an animated Svelte Logo
+
+In Svelte you can add CSS directly to your component. This step demonstrates this capability by adding an animated logo.
+
+[Download `logo.svg`](https://github.com/snowpackjs/snowpack/blob/master/create-snowpack-app/app-template-svelte/public/logo.svg) to your `public` directory. Now you can add it to your `App.svelte`
+
+```diff
+<!-- src/App.svelte -->
+
+  <header class="App-header">
++   <img src="/logo.svg" class="App-logo" alt="logo" />
+    <a
+      class="App-link"
+      href="https://svelte.dev"
+      target="_blank"
+      rel="noopener noreferrer">
+      Learn Svelte
+    </a>
+```
+
+<div class="frame"><img src="/img/guides/svelte/svelte-logo-snowpack.jpg" alt="Side by side of code and site. The site now has a very large Svelte logo. The code shows the src/App.svelte file "  class="screenshot"/></div>
+
+With Svelte, CSS can go directly in your `.svelte` component. Add this code to the top of `App.svelte` between the `<style>` tags:
+
+```html
+<!-- src/App.svelte -->
+
+<style>
+  .App-header {
+    background-color: #f9f6f6;
+    color: #333;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+  }
+  .App-logo {
+    height: 36vmin;
+    pointer-events: none;
+    margin-bottom: 3rem;
+    animation: App-logo-pulse infinite 1.6s ease-in-out alternate;
+  }
+  @keyframes App-logo-pulse {
+    from {
+      transform: scale(1);
+    }
+    to {
+      transform: scale(1.06);
+    }
+  }
+</style>
+```
+
+<div class="frame"><img src="/img/guides/svelte/svelte-logo-style-snowpack.gif" alt="code and site side by side, when the css is added to the Svelte component, the background becomes a beige, the logo shrinks down, and the logo has a pulsing animation" class="screenshot"/></div>
+
+## Adding a counter to your Svelte component
+
+Snowpack is one of the only Svelte dev environments to support Fast Refresh by default. As you make changes to `.svelte` files, Snowpack updates the browser for you without losing your place or resetting component state. That's Fast Refresh in action. To see this for yourself, go ahead and add a simple timer to your App.svelte component.
```

Comment:
Sentence 1 introduces the concept of Fast Refresh, but then doesn't tell you what that is until Sentence 3. Can you make it more clear what Fast Refresh is in the second sentence?